client/components/card/index.jsx

```diff
@@ -5,66 +5,56 @@ import React from 'react';
 import { assign, omit } from 'lodash';
 import classnames from 'classnames';
 import Gridicon from 'gridicons';
-import createClass from 'create-react-class';
 import PropTypes from 'prop-types';
 
-export default createClass( {
-	displayName: 'Card',
+const Card = ( props ) => {
+	const { href, tagName, target, compact, children, highlight } = props;
```

Comment:
Very nice refactoring!

You can avoid the `omit` call by doing:
```js
const { href, tagName, target, compact, children, highlight, ...rest } = props;
```
`rest` is now an object identical to result of `omit(props, omitProps)`

Some Git cleanup might be needed in this branch: it contains the `CreateClass` -> `createClass` commit and also a merge commit from master. A simple `git rebase master` should make it into a branch with just one plain commit.

Reply:
@jsnajdr The omitProps is different depending on whether href is set or not. Is it still possible to use `rest` in that case?

Reply:
@mikkamp Oh, that's a good point. I overlooked that `href` and `target` are used by the Card componen and passed down to the child at the same time. You can still use `rest` then, but then you need to pass `href` and `target` down explicitly:
```js
React.createElement(
  ...
  { ...rest, href, target, className },
  ...
);
```